ghost/stripe/test/unit/lib/services/webhooks/InvoiceEventService.test.js

```diff
@@ -0,0 +1,201 @@
+const chai = require('chai');
```

Comment:
As far as I know we should be using the built in `assert` library for assertions - rather than `should`/`chai`

Reply:
Thanks for the heads up. Swapped them out!